examples/galaxies/plot_size.py

```diff
@@ -0,0 +1,138 @@
+"""
+Galaxy size
+===========
+
+This example demonstrate how to obtain sizes for early and late type galaxies
+in SkyPy.
+
+"""
+
+
+# %%
+# Size-magnitude relation
+# -------------------------
+#
+# According to Shen et al. 2003 [1]_, the observed size of galaxies
+# and the absolute magnitudes follow simple analytic formulae. For early-type
+# galaxies, the relation (14) reads
+#
+# .. math::
+#
+#    log (\bar{R}/kpc) = -0.4aM + b,
```

Comment:
Do we want to be specific about the base of the log? I'm guessing log_10 rather than ln. Only a minor thought.